src/screens/Language/index.js

```diff
@@ -8,34 +10,50 @@ export default function LanguageScreen({ navigation }) {
   const { t, i18n } = useTranslation();
   const { availableLanguages, setAppLanguage } = useContext(LanguageContext);
 
-  const [languageSelected, setLanguageSelect] = useState(false);
-  const handleLanguageSelect = () => setLanguageSelect(true);
-
   const handleLanguageChange = async (language) => {
-    setAppLanguage(language);
-    handleLanguageSelect();
+    await setAppLanguage(language);
+    navigation.navigate('TermsAndConditions');
   };
 
   return (
-    <View>
-      <Text>{t('common.welcome')}</Text>
-      <Text>{t('common.language')}</Text>
-      {availableLanguages.map((item, i) => (
-        <Button
-          key={i}
-          title={item.name}
-          onPress={() => handleLanguageChange(item.lang)}></Button>
-      ))}
-      {
-        languageSelected && (
-          <Button
-            title="Next"
-            onPress={() => navigation.navigate('TermsAndConditions')}
-          />
-        )
-        //TODO: Add popup when user doesn't select a Language and proceed
-        //TODO: Change language of NEXT button when german is pressed
-      }
-    </View>
+    <>
+      <View style={styles.homePageView}>
+        <LanguageSVG style={{ position: 'absolute' }} />
+        <View style={styles.buttonsView}>
+          {availableLanguages.map((item, i) => (
+            <Button
+              title={item.name}
+              titleStyle={{ color: '#000000' }}
+              type="solid"
+              raised
+              containerStyle={styles.button}
+              buttonStyle={{ backgroundColor: '#FFFFFF' }}
+              key={i}
+              onPress={() => handleLanguageChange(item.lang)}
+            />
+          ))}
+        </View>
+      </View>
+    </>
   );
 }
+
+const styles = StyleSheet.create({
+  homePageView: {
+    backgroundColor: '#CADEEE',
+    flex: 1,
+    justifyContent: 'center',
+  },
+  buttonsView: {
+    alignItems: 'center',
+  },
+  button: {
+    width: '80%',
+    height: 84,
```

Comment:
better not to use height, but to apply padding to child, and i see that u have no child cuz it is Button. Buttons seemed very limited, i use touchable opacity ..

Reply:
fixed